src/wasm-lib/kcl/src/std/extrude.rs

```diff
@@ -46,17 +46,87 @@ async fn inner_extrude(length: f64, sketch_group: Box<SketchGroup>, args: Args)
     )
     .await?;
 
+    // TODO maybe this should loop through until it gets a edge_id
+    let first_segment = &sketch_group.value[0];
+    let edge_id = match first_segment {
```

Comment:
This is just grabing the first segment and hoping there's an id on there, because we just need any ide from any edge for `Solid3DGetExtrusionFaceInfo` to work.

But maybe we should loop through and break the loop as soon as we get one to make this more robust?

Reply:
why would the first segement not have an id?

Reply:
actually i can fix this up a bit
like [0] will panic if its empty

Reply:
>why would the first segement not have an id?

That's fair

Reply:
okay fixed i looped thru and failed if sketch group is empty or we dont find an edge id anywhere